random-fu/src/Data/Random/Lift.hs

```diff
@@ -5,7 +5,6 @@ module Data.Random.Lift where
 import Data.RVar
 import qualified Data.Functor.Identity as T
 import qualified Control.Monad.Trans.Class as T
-import Data.Random.Source.Std
 
 #ifndef MTL2
```

Comment:
Not sure what is going on here and certainly not for this PR but do we really need this conditional?

Reply:
The MTL2 one? Probably not, I haven't looked into why it is necessary as it is not related to this PR. However, I assume it was done for backwards compatibility, which is most likely obsolete now and can be removed.

Reply:
https://github.com/haskell-numerics/random-fu/issues/72